pilot/helpers/agents/Developer.py

```diff
@@ -465,11 +465,11 @@ def continue_development(self, iteration_convo, last_branch_name, continue_descr
             user_description = 'Can you check if the app works please? ' + user_description
 
             if self.run_command:
-                if self.project.ipc_client_instance is None or self.project.ipc_client_instance.client is None:
+                if self.project.check_ipc():
```

Comment:
logic is same as before just instead of checking
`if not self.project.check_ipc():`
I swapped it around so that we are checking 
`if self.project.check_ipc():`